Feature/tf df serving (#21)
ml6team/quick-tips: structured_data/README.md

This document contains an overview of the currently available labeling tools for time series. What tools are the ones to be used, which ones you probably better avoid.
- [_Tabular Data: Tensorflow Data Validation_ (30/06/2021)](2021_06_30_tensorflow_data_validation):
This notebook showcases the functionality of TensorFlow Data Validation (TFDV), a tool developed by tensorflow
to track anomalous changes in your data.

> - [_TF Decision Forest: Serving TF-DF models with TF Serving_ (02/07/2021)](2021_07_02_TFDF_serving):

Reviewer: ```suggestion
- [_Tabular Data: Serving decision forests with Tensorflow Serving_ (02/07/2021)](2021_07_02_TFDF_serving):
```

>   This Readme documents the use of the custom build TF Serving docker build to serve TF-Decision Forest

Reviewer: ```suggestion
  This Readme documents the use of our custom TF Serving docker image built to serve Tensorflow Decision Forests.
```

>   models.

Reviewer: ```suggestion
```
